integrationtests/Paket.IntegrationTests/ResolverSkipsConflictsFastSpecs.fs

```diff
@@ -38,24 +38,6 @@ let ``#2289 Paket 4.x install command takes hours to complete``() =
     nunitVersion
     |> shouldBeSmallerThan (SemVer.Parse "3.0")
 
-[<Test; Flaky>]
-let ``#2294 Cannot pin NETStandard.Library = 1.6.0``() =
```

Comment:
I feel like both of these are good tests, we should somehow keep them or set them to ignore.
In fact I think if they fail this indicates a regression in the resolver unable to find a solution in the specified test-timeout.
So what looked like "flaky" might just be a perf regression here?

Reply:
On the other hand, yes resolving this might have become harder with the release of new packages.

Reply:
I don't think it was a regression - we fixed a resolver bug and it that made it slower. shit happens

Reply:
>  we fixed a resolver bug and it that made it slower

Isn't that the definition of a perf regression ;)
Anyway I think getting the CI green again is more important than these tests, so fine by me.

Reply:
no. it never was faster - it was just wrong